test/v2/fixtures/getCalculation/allowancesAndDeductions/detail/ReliefsFixtures.scala

```diff
@@ -104,7 +104,10 @@ object ReliefsFixtures {
       |				"allowableAmount": 1000,
       |				"amountUsed": 1000,
       |				"rate": 2
-      |			}]
+      |			}],
+      |   	"topSlicingRelief": {
+      |			  "amount": 2000.58
+      |		  }
```

Comment:
formatting just looks a bit weird for me here, would it not look better to line top slicing relief up with the earlier objects? something kinda more like this?

```suggestion
      |			}],
      |   	                "topSlicingRelief": {
      |                                     "amount": 2000.58
      |		       }
```